classes/Hook.php

```diff
@@ -62,7 +62,7 @@ class HookCore extends ObjectModel
         'table' => 'hook',
         'primary' => 'id_hook',
         'fields' => array(
-            'name' => array('type' => self::TYPE_STRING, 'validate' => 'isHookName', 'required' => true, 'size' => 64),
+            'name' => array('type' => self::TYPE_STRING, 'validate' => 'isHookName', 'required' => true, 'size' => 255),
```

Comment:
```suggestion
            'name' => array('type' => self::TYPE_STRING, 'validate' => 'isHookName', 'required' => true, 'size' => 191),
```
For utf8mb4 :roll_eyes:

Reply:
I don't suppose anyone in their right mind would use emojis + long hook names?!

Reply:
People like using emojis everywhere right now :(

Reply:
Ok as far as I can tell the 191 limit applies for indexed columns regardless of their content, so I'll have to change it.